examples/resnet/resnet_ptq_cpu.json

```diff
@@ -46,7 +46,14 @@
                     ],
                     "user_config":{
                         "user_script": "user_script.py",
-                        "data_dir": "data",
+                        "data_dir": {
```

Comment:
should we create two versions of this json (one with local data and another with aml data)? So that we have a resnet example that works for users with no access to the datastore.

Reply:
Agree. Will do it.